src/watchful/enrich.py

```diff
@@ -275,6 +287,9 @@ def main(
             # of the dataset. ``spacy_atterize_fn``, ``spacy_atterize`` and
             # ``load_spacy()`` are the additional user variables to perform the
             # data enrichment.
+            if check_import("spacy") is False:
```

Comment:
Hm, I think I'd rather see an attempt to import here and catch the `ImportError` rather than have the function (importing `importlib` is not for the kid's table).

Reply:
Ok. I'll make that change within the `check_import` method rather than duplicate the logic for both spacy and flair.